js/stable.js

```diff
@@ -585,21 +583,18 @@ dxSTable.prototype.assignEvents = function()
 	this.scOdd = null;
 	this.isScrolling = false;
 
-	$(this.dBody).on( "scroll",
-		function(e) 
-		{
+	$(this.dBody).on("scroll", function(e) {
```

Comment:
We're performing multiple calculations on every scroll. Does it make sense to debounce this?